packages/ui/src/documents/CreateDocument.tsx

```diff
@@ -83,13 +83,14 @@ export const CreateDocument: FunctionComponent<Props> = (props) => {
       defaultDocument: document,
     });
       try {
+
         push(documentRoutes.index);
-        go(0);
         if (document.template && document.template !== '') {
           await httpClient.documents.clone(document);
         } else {
           await httpClient.documents.create(document);
         }
+        go(0);
```

Comment:
I think this was the problem with the proxy error. Also we were not getting the alerts from the document errors. We can improve this for sure but I think it is a different topic